src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -97,6 +97,10 @@ public function load(array $configs, ContainerBuilder $container)
             if (is_dir($dir = $bundle['path'].'/Resources/views')) {
                 $this->addTwigPath($twigFilesystemLoaderDefinition, $dir, $name);
             }
+
+            if (null !== $bundle['parent']) {
+                $this->prependParentPaths($twigFilesystemLoaderDefinition, $container, $name, $bundle['parent']);
```

Comment:
@stof: Did not now about that. That'll require some change to adding the normal paths too, I'll see how to best fix this. Is there an easy way to cover this in the test?